### DISCUSSION TITLE
"UNKNOWN: unknow error, read", but still work

### DISCUSSION THREAD

**Discussion by Huiran03** [remotely-save/remotely-save#806] [Q&A] [answered]
Nice plugin! I changed the service from OneDrive to D3, and the info "UNKNOWN: unknow error, read" jump out. but the save process going well. if this is just a display bug?  Should I notice it and find way to resolve it?

and this is its display

![this is what happed](https://huiran-pic-bucket-1308407227.cos.ap-shanghai.myqcloud.com/obsidian/20240911173424.png)

**Comment by fyears** [comment, marked as answer]
你好，你是不是用了非常旧的版本。可以尝试 **备份vault之后** 更新最新版试试，

**Reply by Huiran03** [reply]
谢谢你，这个问题已经在更新插件后得到了完美解决
